### PR TITLE
[CFM-1290] Handling leaving a group (feedback)

### PR DESCRIPTION
**Pull request by DavidHernandez**
Issue: https://issuetracker.amplexor.com/jira/browse/CFM-1290

Issues to be fixed:
1. after leaving the group is complete, the membership should not be exists anymore.
2. after complete, redirect to the group homepage.
3. replace "member" with "group admin" on the tooltip.
4. if there are no admins on the group then don't display the dropdown, and add a notice about it.

The first issue is already working properly. Screenshots of the process:
### Logged in as Galileo, leaving group Nobel Prize:
galileo-left
### Confirmation message:
galileo-confirm:
### Redirected to the homepage (issue 2).
galileo-gone
### List of users in the group:
galileo-list

### Third issue:
admin-tooltip

### Fourth issue:
admins-not-found

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module [hunk 1]
```diff
@@ -194,28 +194,33 @@ function c4m_og_replace_owner_build_form($form, $form_state, $entity_type, $etid
...
+      '#value' => $entity_type,
+    );
+
+    $form['submit'] = array(
+      '#type' => 'submit',
+      '#value' => t('Replace'),
+    );
+  }
+  else {
+    drupal_set_message(t('The group does not have any site admin to be the new owner.'));
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module]: @DavidHernandez 
This probably should be an error / warning and not a "green" message.

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module]: also, can you please add a link at the end to the people page so the flow will be easier ?

**Comment by DavidHernandez** [comment]
replace-owner-warning

**Comment by ordavidil** [comment]
### Dependent on #1149

**Comment by ordavidil** [comment]
@DavidHernandez 
please re-roll

**Pull request merged**